content/03.prior-knowledge.cascadelearning.md

```diff
@@ -0,0 +1,14 @@
+## Cascade Learning {.page_break_before}
+
+Sometimes, due to paucity of data and its unstable predictive capabilities, one may feel the need to consult various machine learning methods and the predictions there-of to make an educated decision about the identified predictive patterns. 
+This assembly of multiple machine learning methods is called ensemble learning. In an ensemble learning workflow, one may use various parameters to maximize the learned information. 
+Sometimes a consensus approach is ideal to select predictive features from a group of independently running models, other times one may coordinate various learning methods in such a way that one model uses information from another model in augmenting its predictive capability. 
```

Comment:
I would change this to "A consensus approach is often...." to avoid this sentence starting with Sometimes so soon after another one